firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -119,21 +107,47 @@ class CertificatesViewController: UIViewController, Themeable, UITableViewDelega
         ])
     }
 
-    private func setupCertificatesHeaderView() {
-        // TODO: FXIOS-9834 Add tab indicator for table view tabs and hide/unhide it on selection
+    private func setupCertificatesTableView() {
+        certificatesTableView.delegate = self
+        certificatesTableView.dataSource = self
+        view.addSubview(certificatesTableView)
+        NSLayoutConstraint.activate([
+            certificatesTableView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor),
+            certificatesTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            certificatesTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            certificatesTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+
+    // MARK: - UITableViewDataSource
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard !viewModel.certificates.isEmpty else { return 0 }
+        return 4
```

Comment:
It is better not to use a constant but to let the data decide.

You can add the `CaseIterable` protocol to `CertificatesItemType`. Then here, you can do the following:
```suggestion
        return CertificatesItemType.allCases.count
```